test/desktop.spec.js

```diff
@@ -76,12 +76,12 @@ describe('Using monkey on desktop', function () {
       });
     });
 
-    it('should not make hyphens in names clickable', function () {
-      $container.find('.nonclickable.special-char .char:contains(-)').length.should.equal(1);
+    it('should not include hyphens in names', function () {
+      $container.find('.char:contains(-)').length.should.equal(0);
```

Comment:
Hyphens disappearing from names is definitely a bug. It's how I originally wrote it, but there were too many complaints so I added the hyphens back.